tests/socket.rs

```diff
@@ -112,6 +114,14 @@ fn socket_address_unix() {
     assert!(addr.as_socket_ipv6().is_none());
 }
 
+#[test]
+#[cfg(all(feature = "all", any(target_os = "android", target_os = "linux")))]
+fn socket_address_vsock() {
+    let addr = SockAddr::vsock(1, 9999).unwrap();
+    assert!(addr.as_socket_ipv4().is_none());
+    assert!(addr.as_socket_ipv6().is_none());
```

Comment:
Can you add `assert!(addr.vsock_port(), Some(9999));`

Reply:
done, thanks for the suggestion